oauth2_provider/models.py

```diff
@@ -859,3 +913,15 @@ def is_origin_allowed(origin, allowed_origins):
             return True
 
     return False
+
+
+def send_backchannel_logout_requests(user):
```

Comment:
per the comment on id_token.send_backchannel_logout_request, let move this to the handlers.py...